config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   post "/login", to: "sessions#create"
-
-  resources :user
+  delete '/logout', to: "sessions#destroy"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.